.github/workflows/deploy-dotorg.yml

```
@@ -2,9 +2,7 @@ name: Deploy to WordPress.org
 
 on:
   release:
-    types:
-      - released
-      - prereleased
+    types: [published]
```

Nit-pick, but maybe better to use the expanded syntax for consistency with the rest of our workflows.
```suggestion
    types:
      - published
```

---

Fixed. ✅